Updated project status.

MartinMueller2003 opened the pull request:
Updated the readme

forkineye commented:
For some reason, the compiled version of esptool for windows was crashing on me.  I ended up grabbing the python embedded binary that they provide for windows (didn't know it existed) and modified ESPSFlashTool to use python for all platforms now.  The changes are in the ESPSFlashTool legacyV3 branch - https://github.com/forkineye/ESPSFlashTool/tree/v3legacy, but I'll do the same to main soon and bring it over here.

MartinMueller2003 commented:
Hi Shelby
I modified the code to use LittleFS. Updated the readme to point at versions of the uploaders that support little FS. Also added a not that lib manager must be used to bring in LITTLEFS-32 for the ESP32.
I will leave flashtool alone until you are done with it.
Martin
    On Tuesday, November 3, 2020, 04:14:30 PM EST, Shelby Merrick <notifications@github.com> wrote:  
 
 


For some reason, the compiled version of esptool for windows was crashing on me. I ended up grabbing the python embedded binary that they provide for windows (didn't know it existed) and modified ESPSFlashTool to use python for all platforms now. The changes are in the ESPSFlashTool legacyV3 branch - https://github.com/forkineye/ESPSFlashTool/tree/v3legacy, but I'll do the same to main soon and bring it over here.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
